src/common/tests/CommonUtilsUT.cpp

```diff
@@ -1002,170 +1002,25 @@ TEST_F(CommonUtilsTest, DuplicateString)
     FREE_MEMORY(duplicate);
 }
 
-TEST_F(CommonUtilsTest, PersistentHashString)
+TEST_F(CommonUtilsTest, HashCommand)
 {
-    EXPECT_EQ(nullptr, PersistentHashString(nullptr, nullptr));
+    EXPECT_EQ(nullptr, HashCommand(nullptr, nullptr));
 
     char* hashOne = nullptr;
     char* hashTwo = nullptr;
     char* hashThree = nullptr;
     
-    const char testOne[] = "This is a test 1234567890";
-    const char testTwo[] = "This is a test 1234567890 test";
+    const char testOne[] = "echo \"This is a test 1234567890\"";
+    const char testTwo[] = "echo \"This is a test 1234567890 test\"";
 
-    EXPECT_NE(nullptr, hashOne = PersistentHashString(testOne, nullptr));
-    EXPECT_NE(nullptr, hashTwo = PersistentHashString(testTwo, nullptr));
-    EXPECT_NE(nullptr, hashThree = PersistentHashString(testOne, nullptr));
+    EXPECT_NE(nullptr, hashOne = HashCommand(testOne, nullptr));
+    EXPECT_NE(nullptr, hashTwo = HashCommand(testTwo, nullptr));
+    EXPECT_NE(nullptr, hashThree = HashCommand(testOne, nullptr));
 
     EXPECT_NE(hashOne, hashTwo);
     EXPECT_STREQ(hashOne, hashThree);
```

Comment:
Since `sha256sum` will give the same output each time for a given string, would it be possible to also check the hash returned by `HashCommand()` here?

Reply:
Are we 100% sure the sha356sum will always give the same output for same input on all platforms where this test will be run on? We've been burned before assuming constant responses.